messaging.rst

```diff
@@ -263,13 +263,14 @@ Mediated Transfer
 
 A :term:`Mediated Transfer` is a hash-time-locked transfer. Currently raiden supports only one type of lock. The lock has an amount that is being transferred, a :term:`secrethash` used to verify the secret that unlocks it, and a :term:`lock expiration` to determine its validity.
 
-Mediated transfers have an :term:`initiator` and a :term:`target` and a number of hops in between. The number of hops can also be zero as these transfers can also be sent to a direct partner. Assuming ``N`` number of hops a mediated transfer will require ``6N + 8`` messages to complete. These are:
+Mediated transfers have an :term:`initiator` and a :term:`target` and a number of hops in between. The number of hops can also be zero as these transfers can also be sent to a direct partner. Assuming ``N`` number of hops a mediated transfer will require ``10N + 14`` messages to complete. These are:
 
 - ``N + 1`` mediated or refund messages
 - ``1`` secret request
-- ``N + 1`` secret reveal
-- ``N + 1`` secret
-- ``3N + 4`` ACK
+- ``N + 2`` secret reveal
+- ``N + 1`` unlock
+- ``2N + 2`` processed (for mediated or refund messages, and unlock messages)
+- ``5N + 7`` delivered
```

Comment:
I created an issue about adding links to the terminology definitions https://github.com/raiden-network/spec/issues/162.

`processed` and `delivered`: before this commit, the document contained `ACK`.  I'm just updating that.  It might make sense to separate descriptions for HTL transfer messages and the ACK-like messages (I'm not sure). That would be in a separate PR anyway.

Reply:
So nothing left for this PR, I believe.